vscode/microsoft-kiota/src/utilities/progress.ts

```diff
@@ -0,0 +1,17 @@
+import * as vscode from "vscode";
+
+import { treeViewFocusCommand } from "../constants";
+
+function openTreeViewWithProgress<T>(callback: () => Promise<T>): Thenable<T> {
```

Comment:
here we're duplicating the code, could we refactor the other references to that method so we avoid a temporary duplication please?

Reply:
Missed it.. lemme point the references